TODO.md

```diff
@@ -15,3 +15,17 @@
 - Combine rolls + rerolls?
 - Non-removal Kickoff result values?
 - Include expected value of armor/injury rolls in rolls that could result in KDs?
+
+
+------------------------------------------------------------------------------------
+
+1. Rework base values for ball handling and hitting (and make sure they are including probabilities as param), make sure they have common denominator
+2. Handle dodge/BT/Sure Hands skills properly
+3. MB/CLAW/TACKLE proper handling on hitting
+4. Maybe rework pushes value based on skills of player? If he has CLAW/MB/PO/TACKLE - pushes are wasting a lot of TV?
```

Comment:
I think this actually ties in to a TODO I had above about merging block/armor/injury rolls together, because it lets you model the expected value of blocking with CLAW-POMB by accounting for likelihood of removal as a result of the block.